samples/drivers/uart/passthrough/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(passthrough)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS on)
+zephyr_compile_options(-fdiagnostics-color=always)
```

Comment:
Why?

Reply:
No reason, removed. (I copied the `native_tty` sample)

Reply:
Interesting, will post a PR to remove that